components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/ActionManagementServiceImpl.java

```diff
@@ -80,6 +109,19 @@ public List<Action> getActionsByActionType(String actionType, String tenantDomai
                 IdentityTenantUtil.getTenantId(tenantDomain));
     }
 
+    /**
+     * Update an action of specified type in the given tenant.
+     * This method performs an HTTP PATCH operation.
+     * Only the non-null and non-empty fields in the provided action model will be updated.
+     * Null or empty fields will be ignored.
+     *
+     * @param actionType   Action type.
+     * @param actionId     Action ID.
+     * @param action       Action update model.
+     * @param tenantDomain Tenant domain.
+     * @return
```

Comment:
Missing the description

Reply:
Updated with 487a8d524e3dd6f8618595b7b9d9266bd19f8c59